Samples/CSharp/FieldsExamples.cs

```diff
@@ -1,5 +1,8 @@
-// Copyright (c) 2019 Henry Gabryjelski
-namespace My
+// <copyright file="FieldsExamples.cs" company="PROJECT-AUTHOR">
+// Copyright (©) 2020 PROJECT-AUTHOR. All Rights Reserved
```

Comment:
I recommend removing the year from being explicitly included.

Reply:
Removed